docs/topics/install.txt

```diff
@@ -221,8 +219,8 @@ latest bug fixes and improvements, follow these instructions:
    ``django-admin`` utility command available. In other words, you're all
    set!
 
-When you want to update your copy of the Django source code, just run the
-command ``git pull`` from within the ``django`` directory. When you do this,
-Git will automatically download any changes.
+When you want to update your copy of the Django source code, run the command
+``git pull`` from within the ``django`` directory. When you do this, Git will
+automatically download any changes.
```

Comment:
I'd drop the "automatically" here - it's not automatic, we just manually triggered it!